resources/views/pages/contract/index.blade.php

```diff
@@ -40,27 +40,27 @@
     </section>
 
     <br />
-    <div id="company" class="container">
-    <div class="row" id="company-div">
-        @foreach ($companies as $company)  
-          <a href="{{ route('contractors.single', ['company' => $company->shortname()]) }}">
+    <div id="contractor" class="container">
+    <div class="row" id="contractor-div">
+        @foreach ($contractors as $contractor)  
+          <a href="#">
```

Comment:
we should link to the single page for each contractor right?

Reply:
Yea - I have handled that already, will push a commit